packages/sanity/src/desk/panes/document/DocumentOperationResults.tsx

```diff
@@ -51,10 +51,10 @@ export const DocumentOperationResults = memo(function DocumentOperationResults()
         closable: true,
         duration: 30000, // 30s
         status: 'error',
-        title: getOpErrorTitle(event.op),
+        title: t(getOpErrorTitle(event.op), {op: event.op}),
```

Comment:
is this a good candidate for [context](https://www.i18next.com/translation-function/context)? that way we could write all these as `panes.document-operation-results.operation.error_delete`, `panes.document-operation-results.operation.error_publish`, etc.

Reply:
Oh! Neat! Have we used that anywhere else?

Reply:
Agree with @bjoerge about using context here - see [some preliminary docs here](https://www.notion.so/sanityio/WIP-i18n-documentation-565cab7c1e4a4569a1ae0e3ee19e26c4#7c28e580ed1b4b35a8680d8c63b7a043) - you could also search for `{context: `, eg [this usage](https://github.com/sanity-io/sanity/blob/feat/i18n/packages/sanity/src/core/form/inputs/files/common/PlaceholderText.tsx#L50)

Unfortunate that `event` is not typed, but digging through the types it appears that these are the defined operations:

`'commit' | 'delete' | 'del' | 'publish' | 'patch' | 'discardChanges' | 'unpublish' | 'duplicate' | 'restore'`